core/events/block_events.js

```diff
@@ -247,12 +247,16 @@ Blockly.Events.Create = function(opt_block) {
     this.recordUndo = false;
   }
 
-  if (opt_block.workspace.rendered) {
```

Comment:
Why are we removing this?

Reply:
`blockToDomWithXY` [accepts a Blockly.Block](https://github.com/google/blockly/blob/f94b1db85018b5a7261fa01f7d04cd5c8367fe2c/core/xml.js#L87). This works b/c Blockly.Block has a default implementation of `getRelativeToSurfaceXY` which just returns (0, 0) which is perfectly fine in this case.

It's not a necessary change, so I can remove it from this PR, but I just did the pruning while I was in the area.